### PR TITLE
header coding

### PR DESCRIPTION
**Pull request by Cemalkaralar**
header coding

### CODE REVIEW

**Comment by bermarte** [comment]
The documentation and the check are missing. The commit's text 'bug fixed' is not informative enough, and the description of the issue is not like it should be.

File: styles/assets/logo.svg
Comment (bermarte) [inline styles/assets/logo.svg:1]: Why you have 2 times the same file?

File: styles/index.css [hunk 1]
```diff
@@ -1 +1,100 @@
...
+  display: flex;
+  align-items: center;
+}
+
+.Header-left {
+}
+
+.Header-logo {
+}
+.Header-logo img {
```
Comment (bermarte) [inline styles/index.css:34]: before a CSS rule add an empty line like you did for the previous ones.

File: styles/index.css [hunk 2]
```diff
@@ -1 +1,100 @@
...
+  height: 51px;
+  object-fit: contain;
+}
+
+.Header-right {
```
Comment (bermarte) [inline styles/index.css:39]: Here it's mentioned 'Header-right' but the formatting is just general, 
```
display: flex;
align-items: center;
```
no elements on the right, which is a bit odd.

File: index.html [hunk 3]
```diff
@@ -4,9 +4,46 @@
...
     <link rel="icon" type="image/svg+xml" href="./client/assets/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
     <title>project!</title>
 
     <link href="./styles/index.css" rel="stylesheet" />
   </head>
-  <body></body>
+
+  <body>
```
Comment (bermarte) [inline index.html:13]: Add comments to divide the different parts, for example:
```
<!-- hero -->
...
<!-- main -->
...
<!-- footer -->
...
```

File: index.html [hunk 4]
```diff
@@ -4,9 +4,46 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="./client/assets/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
     <title>project!</title>
```
Comment (bermarte) [inline index.html:8]: choose a better title (:

File: index.html [hunk 5]
```diff
@@ -4,9 +4,46 @@
...
 
     <link href="./styles/index.css" rel="stylesheet" />
   </head>
-  <body></body>
+
+  <body>
+    <header class="Header">
+      <div class="container">
+        <div class="Header-in">
```
Comment (bermarte) [inline index.html:16]: be consistent in naming your CSS classes, you have classes that start with smaller letters and then you used also capital letters.
```
Header
container
```

**Review by bermarte — CHANGES_REQUESTED** [review]
Just check a bit the formatting and run the tests, fix your problems and push again.

File: index.html (hunk 5 above)
Comment (Cemalkaralar) [reply index.html:16]: container is a generic class. and container a css manipulation is not done. but those that start with a capital letter are special classes and include only the section in which they are found. this is a general principle.

File: index.html (hunk 3 above)
Comment (Cemalkaralar) [reply index.html:13]: ok

File: styles/index.css (hunk 2 above)
Comment (Cemalkaralar) [reply styles/index.css:39]: We divided the header as left and right. There is log on the left, menu and button on the right. This code is done to align menu and button. I don't quite understand what's weird.

File: styles/index.css (hunk 1 above)
Comment (Cemalkaralar) [reply styles/index.css:34]: ok

File: index.html (hunk 4 above)
Comment (Cemalkaralar) [reply index.html:8]: :) of course

File: index.html (hunk 5 above)
Comment (bermarte) [reply index.html:16]: It would be better to use small letters.

File: styles/index.css (hunk 2 above)
Comment (bermarte) [reply styles/index.css:39]: It's better to use another name, more general. Because you can reuse it somewhere else.

File: styles/index.css [hunk 6]
```diff
@@ -1 +1,100 @@
...
+}
+
+.container {
+  width: 100%;
+  max-width: 1180px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.Header {
```
Comment (bermarte) [inline styles/index.css:14]: please remove these rules that are empty

**Review by bermarte — APPROVED** [review]

**Pull request merged**